blog/terraform-post.md

```diff
@@ -0,0 +1,132 @@
qyvy/0cgqWgKmrIuTXnu0wlbCRwTGRkZksQ99thjatYawR/AYLNgg/3VVFc+//xzy/n8xRNPPKFmBTaAv9uO7Q68N6o5WPSyDup7pZpvEtnJq/xLl1fguGt0wvC+FlnzF35F0BTcQpeT8q2xY4Wog1afmCXOkK6BZVQskVL3v6lAmHO6ChwATYHtBI7hVqdQJG7BggVq1pDhdW3UMmXwxvA8GLQhCHVfWL4+YG+2ROpyowqPGnJsmaNEl7fykpzAuz+YBG7c589ZZM1fJAYYB+crcTzuLkec31laKZY8UesZKDaOcdRpuzJgH/jzMf6ZWN+N6QOFlmYMBA6AWmNLgWN4cL8qNf6itrNTx40bZylLxvXXX68mrxdCGfPGExpGjRqlZgU2g1sqKsoqQ+p+5FauHbPkBACPWF5EFSxV4FjIflz6oUXW/EXmgeBbZMlyeSKF3vpGIbe8cTewG72npx0b5qdbhc1PyK20AAA1x7YCx4Q6Jo53L6gpPXr0sJQlY9iwYWryOsNrCFUnbxwTJ8o1t4DdYYnLTAi+9IYUuMx4fX203NRqdmkwCVzWoanU497rLcJmjnvuvDbwJAYleLcFXtqE68H1UeupBi/oeyKJ661eOWju8K+8vMhNiz4/4rN5vRoQOABqj60FjgmlO/W+++6jjAzejzJ0gpUZHR2tJq8znTt3tpxHjRdffFHNBpoB1bbCTZBrw1HAPVeN8AqcEC5N4hL3jacrLrvIIm4cV//uFxS3dXRwIVQixzv7lFvVgrXC8WsJq/WmNwjc6U3m0RKKelXvMlW7ViFwANQe2wscU53E8WtxcXFqtqCoZcjgiQ28wX19E2jBYBn9+vVTs4BmAMtNWWHwrtRdc51ia3qPZk2qUFnCJHAcYsmPw1PpoZ43+sjbm690p7Q4faN6SxlBgrtvRb09RJsnWusq5W3rFKdIA3kDTFmhi76bcYwmvQSBA6C+aBYCx3B3aqAFb7t37x50PRiV1atXW8qQcdNNN1W7tk1t6Nq1q+VcMqKiotTkoBnBksNrvvmTOO5eld2n5cVZ1beWKQLHwXlY1Hhiw7H9+mN1M08Dhla+s/ykqM8mPwLH8saTLuR1AaCyYmySkDfeSgsCB0DtaTYCx/CCtv4kjndyqAmvv/66pQwZTz75pJq8Xli5cqXfur/22mtqUtAMYdnhpTY2jLEKkcuhzz7Nz4m2CpUafgSuXkMrPy9rJ/FkCl5GRK0ry5teX/UKAajiRFoZzXnvIAQOgDrQrASOYRFi8eFuU+6WHDp0qJqkWnh9NVWkZCxevFhNXm9MnjxZ1FtGRESEmgQ0Y1h6Sk5VisH/ZinSX6ukzCMLrUKlRkMLnBYZhxcLgcuIdfuMg+Pn+sLDyoUBEICYdQ23niYAzZ1mJ3AS3se0tgSbDdqQC/hKeNswcHrC48aOx7hFdyp3n3KI45oVpewfb5EpSzSCwKXGzRCSVpjlMeSNWw4ryiFvAADQWDRbgastDofDIm3m4AV+AWhIWILcLu8kAWMGaiUl74uwyJQlGkHgjsVO0etYUSnkbfMkB/HOdZA3AABoPCBwCjzDVJU2cwDQGLAMcWydKhfwraSU6DFWmVKjUQRuqqgbb05/bKcLs00BAKAJgMApVNcCB0BjISROPmqWlHZgtlWm1GhogdPKTz84XwilbHWDvAEAQOMDgfMDb7+lihvPEK3JUiQA1CcsTI7SXEreNy748h8NJXAx+nIkSXsjyFmeL+oDAACg6YDA+YFvTr169RKL9vJM1i5dutChQ4fUZAA0KqLVy+2irMSlYnN5IXKqsKk/1zFSOHgduX1jxS4MHo8L8gYAAGEABC4IHo9HBG5YIBzgz6EM7rdMPzBXEzllZmo9CRxvXM+P3OKWn72Xz+57fgAAAE0KBA4Am8DeVFKQpD3qOzPIKC/J0mQrSkiX31a5EIP3WdV3bZggJipwuebz8Di8nJQ1EDgAAAgDIHAA2AQWp5zkNUKweC228uLjxnEZ3MVZ4SgSrWbZicspLWG66G5N2juSkvaYYu8oTdjGUlr8dMpN/YGKTh2hCmeRlp/l0Fymm/Jz9mlCN02c11VRDIEDAIAwAAIHgI3gLn2eyKB3c3KL2STKSlxJjtJs8bqvfOmtZixhLGYsdzIqhahx6EMEzMHdpeUlGZR5dHlVi5xo3ZsAeQMAgDABAgeAjWDhOpH2gzFGzej+ZMGKnkhp8XPo1PEtVOEs0CStoiqfyMuCpj/qR3TcLgc5y3LpVMZWLf8soyvWPNuVyz6RtgECBwAAYQIEDgAbIVvVkqPHWsawGeGVu5SYSNF6xhMdkoWMRWkxRTyKY1qkaK9LGQy0PAkfT42foVYFAABAEwKBA8Bm6GPd3FWTFvzIW32FKN+78wJa3wAAIHyAwAFgQ1imik4dMmaOGtJVjwLHgsgteB63A/IGAABhBgQOAJvCUlWSd9TbDeqVuHoSODEGLno8uV3lkDcAAAhDIHAA2BiWK32LrTG6vNWDwEl5c5SegLwBAECYAoEDwOZIycrP3keJe0Zax8WFGNxlmrh7OJUXZ/qUCwAAIPyAwAHQTGDfcjmLKS9rlxA5uQG9Kmq+0sZj6CZo4vYF5WXuJldFCcQNAABsAAQOgGaEvsxIpRC5koIUyjr6jSZzw4XQ8Yb0Mo5qwpa0dzhlJi6l8uIs8ridRl4AAADhDwQOgGaIlDERHje5KkqpQpM6GW5XGal7qgIAALAPEDgAAAAAAJsBgQMAAAAAsBkQOAAAAAAAmwGBAwAAAACwGRA4AAAAAACbAYEDAAAAALAZEDgAAAAAAJsBgQMAAAAAsBkQOAAAAAAAmwGBAwAAAACwGRA4AAAAAACbAYEDAAAAALAZEDgAAAAAAJsBgQMAAAAAsBkQOAAAAAAAmwGBAwAAAACwGRA4AAAAAACbAYEDAAAAALAZEDgAAAAAAJsBgQMAAAAAsBkQOAAAAAAAm/H/fM0QEembecwAAAAASUVORK5CYII=
```

Comment:
nit: I feel like this should just be static content that we have embed into the markdown, rather than base64 encoded.

Reply:
We don't currently have the code in place to move a static image file to the right place on the site. We can file a bug.